### PR TITLE
refactor to reflect new name

### PR DESCRIPTION
**Pull request by thurber**

### CODE REVIEW

File: mosartwmpy/mosartwmpy.py [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
-class Mosart(Bmi):
-    # Mosart Basic Model Interface
+from mosartwmpy.config.setup import setup
+from mosartwmpy.grid.load_grid import load_grid
+from mosartwmpy.output.output import initialize_output, update_output, write_restart
+from mosartwmpy.state.initialize_state import initialize_state
+from mosartwmpy.update.update import update
+
+class Model(Bmi):
+    # MosartWmPy Basic Model Interface
```
Comment (crvernon) [inline mosartwmpy/mosartwmpy.py:17]: Need docstring here that also describes what you are passing into __init__ as well as describing the purpose of the class

File: validation/month_validate.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from xarray import open_dataset
 vars = ['RIVER_DISCHARGE_OVER_LAND_LIQ', 'STORAGE_LIQ', 'QSUR_LIQ', 'QSUB_LIQ', 'WRM_DEMAND', 'WRM_DEMAND0', 'WRM_STORAGE', 'WRM_SUPPLY']
 b = open_dataset('validation/wm-validation-1981-1982.nc')
-p = open_dataset('output/WM_Validation/WM_Validation_1981_01.nc')
+p = open_dataset('output/Fiddling/Fiddling_1981_01.nc')
```
Comment (crvernon) [inline validation/month_validate.py:6]: If this data is going to be part of the package, you should access it via `pkg_resources` and include it in a `MANIFEST.in` file and add an include package data arg to the setup function in your `setup.py`

**Review by crvernon — COMMENTED** [review]
Looking great @thurber ! Just a few comments. This is from my phone so I spared the excess. Let me know if a comment needs further explanation. Thank you!

File: mosartwmpy/mosartwmpy.py (hunk 1 above)
Comment (thurber) [reply mosartwmpy/mosartwmpy.py:17]: Makes sense -- i'll make a separate effort to add all the docstrings

File: validation/month_validate.py (hunk 2 above)
Comment (thurber) [reply validation/month_validate.py:6]: Yes, i'll need to make these validation utilities more robust, and include validation data as you mention -- will address in a separate effort

File: mosartwmpy/mosartwmpy.py (hunk 1 above)
Comment (thurber) [reply mosartwmpy/mosartwmpy.py:17]: #5

File: validation/month_validate.py (hunk 2 above)
Comment (thurber) [reply validation/month_validate.py:6]: #6

**Pull request merged**